### ISSUE TITLE
Populate SERIAL (auto_increment) primary key in RETURN after CREATE

### ISSUE THREAD

**Issue by sapalli2989** [kuzudb/kuzu#2167]
With `SERIAL` AUTO_INCREMENT type, it is currently difficult to retrieve the ID of created records, as it is *not* `RETURN`ed as part of the create statement. An additional query is needed (N+1 query problem) and records needs to be discovered by re-querying over its properties.

Example: Given table
```cypher
CREATE NODE TABLE Test(id SERIAL, content STRING, PRIMARY KEY(id));
```
only internal `_ID` is returned, not the `SERIAL` key with name `id`:
```cypher
kuzu> CREATE (t:Test {content: "mycontent"}) return t;
------------------------------------------------
| t                                            |
------------------------------------------------
| {_ID: 0:0, _LABEL: Test, content: mycontent} |
------------------------------------------------
```
The auto_increment `id` is only available with subseqeuent  queries:
```cypher
kuzu> match (t:Test) return t;
-------------------------------------------------------
| t                                                   |
-------------------------------------------------------
| {_ID: 0:0, _LABEL: Test, id: 0, content: mycontent} |
-------------------------------------------------------
```
Suggestion: Either populate `SERIAL` key directly with `RETURN` in the `CREATE` statement.

Or make internal `_ID` publicly accessible as query column, so we can first return it on create (already works as of today):
```cypher
kuzu> CREATE (t2:Test {content: "mycontent2"}) return id(t2);
----------
| t2._ID |
----------
| 0:1    |
----------
```
and then somehow be able to find a record by its internal ID, for example:
```cypher
MATCH (t:Test) WHERE t._ID = '0:1' return t;
```
(this doesn't work yet)

Related: 
- [Return After Update](https://kuzudb.com/docusaurus/blog/kuzu-0.0.7-release/#return-after-update)
- [SERIAL type](https://kuzudb.com/docusaurus/cypher/data-types/serial/)

**Comment by ray6080** [comment]
We should return `SERIAL` key with `RETURN` in the `CREATE` statement. This looks like a bug to me. Will fix it soon!

**Comment by andyfengHKU** [comment]
Solved in #2181
